docs/uml/sequenceDiagrams/plans/createPlan-Part2.puml

```diff
@@ -0,0 +1,76 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+!include ../../Style.puml
+
+autonumber 7.1
+
+title Sequence Diagram of `PlanList#createAndAddPlan()`
+
+participant "planList\n:PlanList" AS pl
+participant "workoutList\n:WorkoutList" AS wl
+
+activate pl
+group sd Write new Plan data into file on user's system.
+
+ref over pl
+Parse the user input to obtain the plan name
+and the workout index numbers separated by comma.
+(Method calls not shown)
+end
+
+pl -> pl : checkPlanNameValidity(userPlanNameInput, className)
+activate pl
+return
```

Comment:
For self-invocations, maybe consider using `deactivate p1` here instead of `return`, such that there isn't a returning line shown in the sequence diagram? In my opinion, the return line doesn't convey much extra information and might make the diagram look a little cluttered, especially since each line is labeled.